src/Components/ContactsListComponents/ContactsList.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { View, FlatList } from 'react-native';
+import EveryContact from './EveryContact';
+
+const allContactsData = [
+  {
+    name: 'Aron Ashluxe',
+    number: '(907) 555-0101',
+    id: Math.random(),
+    photo: '',
+  },
+  {
+    name: 'Annette Black',
+    number: '(907) 555-0102',
+    id: Math.random(),
+    photo: '',
+  },
+  {
+    name: 'Albert Cooper',
+    number: '(907) 555-0103',
+    id: Math.random(),
+    photo: '../ContactsListComponents/ContactsImage/Rectangle60.png',
+  },
+  {
+    name: 'Annette Flores',
+    number: '(907) 555-0104',
+    id: Math.random(),
+    photo: '',
+  },
+  {
+    name: 'Peter Parker',
+    number: '(907) 555-0105',
+    id: Math.random(),
+    photo: '',
+  },
+  {
+    name: 'Johnny Depp',
+    number: '(907) 555-0106',
+    id: Math.random(),
+    photo: '',
+  },
+];
+
+const ContactsList = () => {
+  const renderContacts = contacts => {
+    return <EveryContact contacts={contacts} />;
+  };
```

Comment:
Please do destructuring.  contacts =>  ({ item. index }),
change contacts prop to contact and pass only Item.